x-pack/plugins/ml/public/application/settings/calendars/edit/new_event_modal/new_event_modal.js

```diff
@@ -293,13 +299,18 @@ export class NewEventModal extends Component {
           </EuiModalBody>
 
           <EuiModalFooter>
-            <EuiButtonEmpty onClick={closeModal}>
+            <EuiButtonEmpty onClick={closeModal} data-test-subj={'mlCancelCalendarEvent'}>
               <FormattedMessage
                 id="xpack.ml.calendarsEdit.newEventModal.cancelButtonLabel"
                 defaultMessage="Cancel"
               />
             </EuiButtonEmpty>
-            <EuiButton onClick={this.handleAddEvent} fill disabled={!description}>
+            <EuiButton
+              onClick={this.handleAddEvent}
+              fill
+              disabled={!description}
+              data-test-subj={'mlAddCalendarEvent'}
```

Comment:
To be consistent with other naming, I'd suggest to make this `mlCalendarAddEventButton`.